lib/ozonetel/client.rb

```diff
@@ -0,0 +1,32 @@
+require 'httparty'
+
+module Ozonetel
+  class Client
+    include HTTParty
+    MANUAL_DIAL_URL = "http://cloudagent.in/CAServices/AgentManualDial.php"
+
+    def initialize(customer, api_key, campaign_name)
+      @user_name = customer
+      @api_key = api_key
```

Comment:
Shouldn't api_key be the first parameter? Or is this API key specific to a customer?